framework-plugins/lisk-framework-dashboard-plugin/src/ui/components/Table/Table.module.scss

```diff
@@ -0,0 +1,51 @@
+@import '../../styles/constants';
+
+.table {
+	background: $color-background-darker;
+	width: 560px;
```

Comment:
Please don't fix the width of table, as its a generic presentational component and can be used in different cases. Let it adjust as per container.